### ISSUE TITLE
Add a license to make its open source terms explicit

### ISSUE THREAD

**Issue by grdryn** [redhat-cop/operator-utils#4]
I believe that without being made available under the terms of a given license, code is not really open source. I'll happily create a PR to add it if that saves you some effort, but I'm not going to do anything until I know what license you want to make it available it under. I'd suggest Apache 2.0. :slightly_smiling_face:

**Comment by raffaelespazzoli** [comment]
Yes please, if you could send a pr that would be great. I'm not an expert on the legal side of open source licenses.
